spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -269,8 +269,8 @@ static boolean parseOverallSuccess(SpoonSummary summary) {
       if (result.getInstallFailed()) {
         return false; // App and/or test installation failed.
       }
-      if (!result.getExceptions().isEmpty() && result.getTestResults().isEmpty()) {
-        return false; // No tests run and top-level exception present.
+      if (!result.getExceptions().isEmpty() || result.getTestResults().isEmpty()) {
+        return false; // Top-level exception present, or no tests were run
```

Comment:
nit: missing period on comment.

[Comments should be sentences.][1]

 [1]: http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html